.vscode/tasks.json

```
@@ -46,6 +46,14 @@
       "options": {
         "cwd": "${workspaceFolder}/languages/python"
       }
-    }
+    },
+    {
+      "label": "buildJava",
+      "type": "shell",
+      "command": "gradle",
+      "args": ["build"],
+      "dependsOrder": "sequence",
+      "dependsOn": ["rust: bitwarden-c build"]
+  }
```

This task seems to be missing it's working directory
```suggestion
      "dependsOn": ["rust: bitwarden-c build"],
      "options": {
        "cwd": "${workspaceFolder}/languages/java"
      }
  }
```